app/views/catalog/_index_header_list_default.html.erb

```diff
@@ -1,3 +1,10 @@
+<% model = document["has_model_ssim"].first.safe_constantize %>
```

Comment:
There is a [method for this on SolrDocument](https://github.com/samvera/hyrax/blob/main/app/models/concerns/hyrax/solr_document_behavior.rb#L77-L81) which has more handling of edge cases (primarily wings I believe).
```suggestion
<% model = document.hydra_model %>
```